documentation/puppetdb_connection.markdown

```diff
@@ -70,3 +81,32 @@ The default value is false.
 
 Hostname and port of the PuppetDB instance. `server_urls` takes precedence if both are defined.
 
+### High Availability configuration
+
+These settings are designed for use in a high-availability deployment of PuppetDB, available in Puppet Enterprise. While they may appear to work in other configurations, we discourage their use. In particular, absent the synchronization support in Puppet Enterprise, multiple PuppetDB instances can easily diverge from each other due to normal transient network issues. See the [PuppetDB HA Configuration Guide][ha_guide] for detailed configuration information.
+
+#### `sticky_read_failover`
+
+*PE-only*
+
+When using multiple `server_urls`, this flag can be set to `true` to cause queries to be made to the last PuppetDB instance that was successfully contacted. 
+
+The default value is `false`.
```

Comment:
We'll tell people to change it to true for HA... I don't think the defaults should be different for foss vs. PE.